### PR TITLE
feat - Update footer, community links, and implement dark mode on landing page

### LINKED ISSUE #92: Update Footer and Community Links and Implement Dark Mode on Landing Page
1. Reposition "Powered by Starknet" and Update Community Links
Description: Move the "Powered by Starknet" element from the hero section to the footer. Replace it with the text "Onchain land, decentralized and secured." Additionally, include the Telegram link in the community links and create a dedicated "Guides" page. 

2.  Provide users with the option to switch between light and dark mode on the landing page. This will enhance user experience and accommodate different preferences. 
 

Definition of Completion: A toggle or button is available on the landing page to switch between light and dark mode. The page's appearance adjusts accordingly, ensuring readability and visual comfort in various lighting conditions.

Ref:

https://landver.vercel.app/

https://github.com/NoshonNetworks/landver/tree/main/frontend/landing_page

### PR DESCRIPTION
**Pull request by Eche5**
## Update footer, community links, and implement dark mode on landing page

---
## Related Issues

Closes #92 

---

## Type of Change

- [ ] 🐛 Bug fix or ⚙️ enhancement
- [X] ✨ New feature or Chore (change with no new features or fixes)
- [ ] 📚 Documentation update

---

## Checklist (select as many as applicable)

- [X] The code follows the style guidelines of this project.
- [X] All new and existing tests pass.
- [X] This pull request is ready to be merged and reviewed.

### CODE REVIEW

**Comment by fishonamos** [comment]
Please always fix on original pr for tracking purpose.

**Comment by Eche5** [comment]
@Solomonsolomonsolomon, could you please review this PR?

**Comment by Solomonsolomonsolomon** [comment]
@Eche5 can you please link the issue number to the PR,

> template for linking was provided in the Related issues section  in the PR

**Comment by Eche5** [comment]
> @Eche5 can you please link the issue number to the PR,
> 
> > template for linking was provided in the Related issues section  in the PR



> @Eche5 can you please link the issue number to the PR,
> 
> > template for linking was provided in the Related issues section  in the PR

 could you please re-review

File: landing_page/src/app/components/Footer.tsx [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+            <div className="text-left h-auto md:p-2 lg:p-2">
+              <h2 className="text-xl font-semibold text-gray-800">
+                Powered by StarkNet
+              </h2>
+              <p className="text-sm text-gray-600">
+                Unparalleled security and scalability
+              </p>
+            </div>
+          </div>
+        </div>
```
Comment (Solomonsolomonsolomon) [inline landing_page/src/app/components/Footer.tsx:26]: please centralize this, it is currently aligned to the left

**Review by Solomonsolomonsolomon — CHANGES_REQUESTED** [review]
dark mode not implemented in about page

File: landing_page/package.json
Comment (Solomonsolomonsolomon) [inline landing_page/package.json:1]: running npm run dev gives build issues, fix the issues as app doesnt compile

**Review by Solomonsolomonsolomon — CHANGES_REQUESTED** [review]

**Review by Solomonsolomonsolomon — APPROVED** [review]
LGTM!

**Pull request merged**